src/scss/index.scss

```diff
@@ -1 +1,109 @@
+@import "variables";
+@import "../../node_modules/bootstrap/scss/bootstrap.scss";
 
+$enable-shadows: true;
+
+
+.btn {
+  text-decoration: none;
+}
+
+/* nav bottom */
+.navbar-bottom {
+  background-color: $secondary;
+
+  i.fas {
+    color: $primary;
+  }
+
+  p {
+    color: $primary;
+  }
+}
+
+/* body styles */
+body {
+  background-color: $primary;
+}
+
+.bg-img {
+  //image from: https://www.pexels.com/nb-no/bilde/sjarmerende-om-harmoni-ung-4130215/
+  background-image: url("/img/pexels-elina-sazonova-4130215.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+/* Authentication page */
+
+.navbar-custom {
+  background-color: $primary;
+}
+
+.max-width {
+  max-width: 100%;
+}
+
+/* Log in form */
+.form-container {
+  justify-content: center;
+}
+
+.form-wrapper {
+  .form-floating {
+    input.form-control {
+      background-color: $primary;
+    }
+
+    label {
+      color: $light;
+    }
+  }
+}
+
+.form-title {
+  color: $primary;
+}
+
+/* Profile page */
+.navbar-profile {
+  background-color: $primary;
+  .container {
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .navbar-brand {
+    color: $dark;
+  }
+}
+
+
+
+/* profile container */
+.profile-container {
+  background-color: $primary;
+  border-radius: 0px 0px 70px 70px;
+  width: 100%;
+}
+
+/* Styling for the profile section container */
+
+.profile-section {
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  img.profile-image {
+    width: 200px;
+    height: 200px;
+    border-radius: 50%;
+    object-fit: cover;
```

Comment:
https://getbootstrap.com/docs/5.3/utilities/borders/
https://getbootstrap.com/docs/5.3/utilities/object-fit/

Reply:
thanks, done. Looks like the img is still a bit stretched no matter how I style it with either of the object-fit values, height/width and rounded-circle